### PR TITLE
Fixing solana.latest_balances, adding sns_domains and total_rewards, adding WBTC/JitoSOL

### PR DESCRIPTION
**Pull request by andrewhong5297**
Fixing solana.daily_balances and solana.latest_balances:
- there was an error before where daily_balances is sometimes null so latest would show up as null. This fixes that case.
- it will require a full rerun of the daily_balances incremental table.

Adding sns_domains:
- equivalent of ens domains

Adding total_rewards
- summing across solana.rewards is slow to get total rewards/fees etc. this table makes it faster.

solana.fungible_tokens got one token added:
- WBTC (wormhole)

### CODE REVIEW

File: models/solana_utils/solana_utils_latest_balances.sql [hunk 1]
```diff
@@ -2,6 +2,7 @@
   config(
         alias = alias('latest_balances'),
```
Comment (antonio-mendes) [inline models/solana_utils/solana_utils_latest_balances.sql:3]: Can you add a tag here like  `schema = 'solana_utils'`? It should help with the CI

**Comment by andrewhong5297** [comment]
fyi this will require a full rerun of the daily_balances incremental table.

**Comment by andrewhong5297** [comment]
Ready to go! I've QAed all the tables :)

**Comment by andrewhong5297** [comment]
@antonio-mendes I've QAed all the tables, they're good to go 🖖

**Comment by antonio-mendes** [comment]
This is timing out on SparkSQL, let me check with the engine team if it's safe to merge or if we need to prepare for it and scale up

**Comment by antonio-mendes** [comment]
@andrewhong5297 I cancelled the CI manually on purpose now. The prod job is failing and it shares resources with the CI.

I'll trigger the testing and merge this today but need Spellbook on Spark to be in a good state in prod first

**Comment by antonio-mendes** [comment]
Ok prod job caught up and I'm confident this will run faster in prod than in the CI so I'll bypass it. Will monitor the jobs for the next hours

**Review by antonio-mendes — APPROVED** [review]
lfg

**Pull request merged**